docs/user-guide/class-b-beaconing.md

```diff
@@ -0,0 +1,171 @@
+# Class-B: Beacon locking
+
+This doc describes the process of locking a Class-B arduino device to a beaconing signal issued from the basic station and reading the GPS coordinates transmitted by that beacon.
+
+## Network Server
+The LoRaWAN network server (`LoraWanNetworkSrvModule`) doesn't need any special configuration beyond the usual required launch settings.
+
+> **Note**: For the time being the network server code required to support Class B beaconing lives in the branch `feature/add-beaconing`
+
+## Basic Station / Concentrator
+For this setup, we use a concentrator with a GPS antenna attached.
+
+> **Note**: The GPS data can be faked by the concentrator in then case when no GPS antenna is available. **This has not yet been tested**. See [Class B Beaconing Settings](https://lora-developers.semtech.com/build/software/lora-basics/lora-basics-for-gateways/?url=conf.html), [Creating a FIFO](https://tldp.org/LDP/lpg/node17.html) and [GPS NMEA data](https://www.gpsworld.com/what-exactly-is-gps-nmea-data/#:~:text=Today%20in%20the%20world%20of,and%20match%20hardware%20and%20software.) for more information.
+
+> **Note**: The following steps are not comprehensive and assume some previous knowledge of running the basic station, such as creating the `tc.uri` file, running in "NO-CUPS" mode, etc.
+
+Clone the [basic station repository](https://github.com/lorabasics/basicstation) code onto the Raspberry Pi.
+
+```bash
+git clone https://github.com/lorabasics/basicstation.git
+```
+
+Build the code:
+
+```bash
+cd basicstation
+make platform=rpi variant=std
+```
+
+Modify `station.conf`:
+
+1. Add `"pps": true` under `SX1301_conf`.
```

Comment:
You can link this issue for additional context on the settings https://github.com/lorabasics/basicstation/issues/98#issuecomment-831204980